pandas/tseries/frequencies.py

```diff
@@ -19,30 +19,19 @@
 from pandas.util._decorators import cache_readonly, deprecate_kwarg
 import pandas.tseries.offsets as offsets
 
-from pandas._libs import lib, tslib
+from pandas._libs import tslib
 from pandas._libs.tslib import Timedelta
-from pandas._libs.tslibs.frequencies import (  # noqa
+from pandas._libs.tslibs.frequencies import (  # noqa:F401
```

Comment:
pls don't add a noqa, let's just remove the functions that are causing it

Reply:
if there are public functions that should remain, put them in a separate import and indicate this